R/NDVI.R

```diff
@@ -0,0 +1,57 @@
+#' @importFrom magrittr "%>%"
+
+
+#' @title NDVI by calendar month or lunar month
+#'
+#' @description Summarize NDVI data to monthly or lunar monthly level
+#'
+#' @param level specify 'monthly' or 'newmoon'
+#' @param path specify where to locate Portal data
+#'
+#' @export
+#'
+summarize_ndvi <- function(level = "monthly", path = '~') {
+  NDVI=read.csv(FullPath('PortalData/NDVI/monthly_NDVI.csv', path), na.strings=c("-99"), stringsAsFactors = FALSE)
```

Comment:
Is there a reason for `na.strings=c("-99")`?

Reply:
that was copied straight over from the weather function, so the justification predates me. if it's unnecessary, we can cut it out